docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -540,64 +547,67 @@ spec:
   - pre-adoption-validation
 EOF
 ----
-+
-Wait for the validation to finish.
+
+.. Wait for the validation to finish.
 
 .. Confirm that all the Ansible EE pods reach a `Completed` status:
 +
 ----
 # watching the pods
-watch oc get pod -l app=openstackansibleee
+$ watch oc get pod -l app=openstackansibleee
 ----
 +
 ----
 # following the ansible logs with:
-oc logs -l app=openstackansibleee -f --max-log-requests 20
+$ oc logs -l app=openstackansibleee -f --max-log-requests 20
 ----
 
 .. Wait for the deployment to reach the `Ready` status:
 +
 ----
-oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
+$ oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption --timeout=10m
 ----
++
+[IMPORTANT]
```

Comment:
This could be a separate section, something like:

"Troubleshooting failed pre-adoption validations"